### ISSUE TITLE
The wiki is outdated

### ISSUE THREAD

**Issue by gridcat** [gridcoin-community/Gridcoin-World-Site#6]
Page: https://gridcoin.world/wiki/

Please migrate the wiki pages from the original .us website, as the current pages are outdated.
If we will move 100% of wiki pages to here it is worth to upgrade the link in the footer as well, as atm it points to the .us website.

**Comment by barton2526** [comment]
I am rather confused by this.  The original stated intention of the .world site was to be a _separate_, lighter landing page for Gridcoin marketing and to refer users to the .us site for more in-depth content.  Is the wiki not something we should be keeping separate on the .us site?

**Comment by gridcat** [comment]
I was not aware of the intention. The website looks like atm more like a updated version of the .us website (consider those links in the bottom to the guides and wiki pages). So I made an assumption.

**Comment by gridcat** [comment]
@barton2526 in this case this issue could be resolved I guess.

**Comment by gridcat** [comment]
But there was an opinion to migrate things here
https://teamgridcoin.slack.com/archives/C6U7BQUR1/p1638291623035400

![image](https://user-images.githubusercontent.com/32429858/144917784-5d88181d-42ea-4b82-8971-956313f38f00.png)

**Comment by barton2526** [comment]
The opinion of one developer does not supersede the entire premise of the .world's site's creation

**Comment by gridcat** [comment]
Then feel free to close this issue

**Comment by jamescowens** [comment]
I would prefer linking to the us site for the wiki if it can be done in an attractive way. There is no reason to have to maintain two separate versions of that.

**Comment by barton2526** [comment]
> Then feel free to close this issue

The intention was to leave it open for now for dialogue.  If everyone agrees, we can close it.

**Comment by RoboticMind** [comment]
I would agree that the wiki should probably just be in one place and not duplicated

**Comment by gridcat** [comment]
Theoretically, the problem of maintenance of 2 copies could be solved. It is possible to make the .us site to be a subpackage of the .world repo and use .md files from the subpackage. Will look ugly but will work. Same with guides.
Otherwise, we need to get rid of the wiki page completely and make the links in the footer point to the .us website.

**Comment by gridcat** [comment]
So I guess we are getting rid of the wiki pages here and we will instead change the link to point to the .us website. Is this aligned? I will close this issue then and will add appropriate tasks later.

**Comment by barton2526** [comment]
I would be interested in @jring-o 's opinion
